js/NavMainComponent.js

```diff
@@ -46,9 +51,13 @@ NavMainComponent.prototype.init = function () {
         if ($self.text() === 'Menu') {
             $self.text('Close');
             $self.attr('aria-expanded', 'true');
+            component.$brandingLink.attr('tabindex', '3');
+            component.$primaryNavLinks.attr('tabindex', '3');
```

Comment:
aXe finds them wrong. Elements should not have tabindex greater than zero -https://dequeuniversity.com/rules/axe/1.1/tabindex Is there another reason we use a greater number?

Reply:
aXe is rather strict for this condition, I believe this change to meet the 2.4.3 guideline.

From technique H4: Creating a logical tab order through links, form controls, and objects
> In some cases, the author may want to specify a tab order that follows relationships in the content without following the order of the interactive elements in the code. In these cases, an alternative order can be specified using the tabindex attribute of the interactive element. The tabindex is given a value between 0 and 32767.
https://www.w3.org/TR/WCAG20-TECHS/H4.html

And from F44: Failure of Success Criterion 2.4.3 due to using tabindex to create a tab order that does not preserve meaning and operability
> If tabindex is used, check that the tab order specified by the tabindex attributes follows relationships in the content.
https://www.w3.org/TR/WCAG20-TECHS/F44.html

Reply:
This has also been raised with aXe team: 
https://github.com/dequelabs/axe-core/issues/572